app/classes/importers/photosynthesis_measurement_importer.rb

```diff
@@ -0,0 +1,65 @@
+class PhotosynthesisMeasurementImporter < RowImporter
+
+  def object
+    @pm
+  end
+
+  def read_row(values, logger)
+    @pm = find_or_new({
+      #########################################################################
+      #
+      # The code reader is going to need to do a      CSP -> tree code lookup
+      # All the data we will need to make I9999999 trees work is also in that
+      # table
+      #
+      #########################################################################
+      :leaf_part => find_or_create_leaf_part(values[1]),
+      :pm_type => values[3]
+    })
+    attempt_to_overwrite!(@pm)
+
+    @pm.filename = values[0]
+    @pm.code = values[1]
+    @pm.area_corr = values[2]
+    begin
+      @pm.date = Date.strptime(values[4], "%d/%m/%y")
+    rescue ArgumentError
+      @pm.date = nil
```

Comment:
We don't want to actually store "NA" as text in a time field, a null entry will represent that well enough. As long as the importer is strict with what it accepts, e.g.

    @pm.date = Date.strptime(values[4], "%d/%m/%y")           unless values[4] == "NA"